src/Kiota.Builder/Refiners/CommonLanguageRefiner.cs

```diff
@@ -104,7 +104,7 @@ public abstract class CommonLanguageRefiner : ILanguageRefiner
             var propertyOriginalName = (currentProperty.IsNameEscaped ? currentProperty.SerializationName : current.Name)
                                         .ToFirstCharacterLowerCase();
             var accessorName = (currentProperty.IsNameEscaped ? currentProperty.SerializationName : current.Name)
-                                .CleanupSymbolName()
+                                .CleanupSymbolName(_configuration.PropertiesPrefixToStrip)
```

Comment:
Should we clean the `SerializationName` centrally [here](https://github.com/microsoft/kiota/blob/main/src/Kiota.Builder/KiotaBuilder.cs#L562) like we do the `Name`?

Or leverage [`SymbolName`](https://github.com/microsoft/kiota/blob/main/src/Kiota.Builder/CodeDOM/CodeProperty.cs#L59) and add prefixes there?

Reply:
We should not do anything to the serialization name as it won't match what the Api is expecting otherwise. Let's have the discussion on the confliting names on the issue you've created.